ES/ESdd041.xml

```diff
@@ -364,17 +378,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <profileDesc>
          <particDesc>
             <listPerson>
-               <person><persName ref="PRS14617Zakrestos"/>
-                  <persName ref="PRS14616MalkeaKrestos"/>
-                  <persName ref="PRS10198Yaeqob"/>
-                  <persName ref="PRS7829Petros"/>
-                  <persName ref="PRS4381Gabriel"/>
-                  <persName ref="PRS8550sardaDe"/>
-                  <persName ref="PRS8276SablaWa"/>
-                  <persName ref="PRS5112harbo"/>
-                  <persName ref="PRS7278Muhammad"/>
-            <persName ref="PRS10343Yeshaq"/>
-               </person>
+               <person><persName ref="PRS14617Zakrestos" role="donor"/>
```

Comment:
if you add these roles to the links to these persons above in the record you do not need to repeat them here

Reply:
The roles were not added, I have inserted them now in the file